GVRf/Sample/scene-objects/src/org/gearvrf/sample/sceneobjects/SceneObjectActivity.java

```diff
@@ -30,15 +37,87 @@
     private float yangle = 0;
     private float xangle = 0;
     private long lastDownTime = 0;
+    private WebView webView;
+    private Camera camera;
 
     /** Called when the activity is first created. */
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        mViewManager = new SampleViewManager();
+
+        createWebView();
+        createCameraView();
+
+        mViewManager = new SampleViewManager(this);
         setScript(mViewManager, "gvr_note4.xml");
     }
 
+    private void createWebView() {
```

Comment:
I can see it reflects my previous comments on Insu’s including the UI thread and non-commercial sample url. No further comment for me at this moment.